### PR TITLE
update webhook example to test signatures

### PR DESCRIPTION
**Pull request by bbradshaw**
[ENG-810](https://onfleet.atlassian.net/browse/ENG-810)

After [ENG-809](https://onfleet.atlassian.net/browse/ENG-809) is deployed and webhooks have outgoing signatures, we should update this sample code (and any relevant docs) to make use of the signatures for verification.

### CODE REVIEW

File: api-tools/webhooks/testwebhooks.py [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+
+@app.before_first_request
+def check_for_secret():
+  if "WEBHOOK_SECRET" in os.environ:
+    app.secret = os.environ["WEBHOOK_SECRET"]
+  else:
+    print("\n***  NOTICE: Webhooks are running in unverified mode.  ***")
+    print("To verify webhooks, copy your Webhook secret key from the Onfleet dashboard "
+    "and set it in the environment as WEBHOOK_SECRET before running this script.\n\n"
+    "For example, export FLASK_APP=testwebhooks.py WEBHOOK_SECRET=XXXXXXXXXXXXXXXXXXXXX flask run")
```
Comment (jamesliupenn) [inline api-tools/webhooks/testwebhooks.py]: I would suggest adding this line in the README as well, specifically calling out the usage of the WEBHOOK_SECRET:

`export FLASK_APP=testwebhooks.py WEBHOOK_SECRET=XXXXXXXXXXXXXXXXXXXXX flask run`

**Comment by ahkhan696** [comment]
I'm trying to use onfleet webhook for PHP script, can anyone tell me how can I use webhook script in PHP?

My webhook is going down every 2nd day.

**Comment by jamesliupenn** [comment]
> I'm trying to use onfleet webhook for PHP script, can anyone tell me how can I use webhook script in PHP?
> 
> My webhook is going down every 2nd day.

@ahkhan696 I have responded to hafizabdulhanan@gmail.com regarding your question which I believe is your email address.

**Review by jamesliupenn — APPROVED** [review]

**Comment by tbutler4** [comment]
@jamesliupenn @sojojo I am hoping to do some open source contributions. I mainly use nodejs but had a brief fling with python. Do you have any suggestions on how I can help?

**Comment by jamesliupenn** [comment]
> @jamesliupenn @sojojo I am hoping to do some open source contributions. I mainly use nodejs but had a brief fling with python. Do you have any suggestions on how I can help?

@tbutler4 my best suggestion is to utilize our node-onfleet API wrapper and go from there. You can contribute in many ways including building developer tools or help with the wrappers themselves!

**Comment by jamesliupenn** [comment]
@sojojo I'm going to merge and close this request 👍

**Comment by tbutler4** [comment]
@jamesliupenn I appreciate the input, great advice! Starting asap!

**Pull request merged**